packages/react-baseline-inputs/src/types.ts

```diff
@@ -100,7 +100,7 @@ export interface FieldInputProps {
   wrapper?: boolean;
   theme?: FieldTheme;
   required?: boolean;
-  requiredIndicator?: React.ReactNode;
+  renderRequired?: ({ className }: { className?: string }) => JSX.Element;
```

Comment:
We should make this a dedicated type:

```typescript
export type RenderRequired = (props: { className?: string }) => React.ReactNode;
```

Then, we can change `utils.ts` to say:

```typescript
export const renderRequiredDefault: RenderRequired = props => <span {...props}>*</span>;
```